test/helpers/utils.ts

```diff
@@ -0,0 +1,31 @@
+import { splitSignature } from "@ethersproject/bytes";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { BigNumber, ethers } from "ethers";
+import { packParameters, signAccessToken } from "../../src/utils";
+
+export const generateEAT = async (
+  signer: SignerWithAddress,
+  domain: any,
```

Comment:
Can we type that?

Reply:
Yes we can 👍

Reply:
I think I misunderstood what you meant. I thought you meant the `generateEAT` function wasn't explicitly typing the return type. What did you mean?

Reply:
Ah I was referring to `domain: any` not the return type of generateEAT